src/screens/CurrencySelectModal.tsx

```diff
@@ -623,6 +611,18 @@ export default function CurrencySelectModal() {
     ],
   }));
 
+  useEffect(() => {
+    if (currentChainId !== prevChainId && currencyList[0]?.data) {
```

Comment:
Tip: when adding important checks like this, it's a great thing to do on its own line with a comment so that we know why it's here. Would make this PR a nice crispy one-line change too, which is always good.

```suggestion
    // check if list has items before attempting to scroll
    if (!currencyList[0]?.data) return
    if (currentChainId !== prevChainId) {
```

Reply:
great tip ty king